test/commands/test_pay.rb

```diff
@@ -82,4 +82,30 @@ def test_sends_from_normal_wallet
       assert_equal(Zold::Amount::ZERO, source.balance)
     end
   end
+
+  def test_notifies_about_tax_status
+    FakeHome.new.run do |home|
+      source = home.create_wallet
+      target = home.create_wallet
+      amount = Zold::Amount.new(zld: 14.95)
+
+      class << test_log
```

Comment:
@yegor256 it can only affect this test if I clone `test_log`. As for defining a regular class, there are problems with that, because I'd need to use `#test_log` (remember --quiet?). And that isn't available outside of method context, and you can't define classes in that context. A way is to include `Test` in the test file context, but it doesn't seem like a good idea.